src/v2i-hub/CswPlugin/src/XmlCurveParser.cpp

```diff
@@ -287,8 +287,12 @@ TiDataFrame* XmlCurveParser::ReadRegions(DOMElement* regionsElement)
 	DsrcBuilder::SetStartTimeToYesterday(frame);
 
 	// Set the duration (minutes) to its max value.
+	#if SAEJ2735_SPEC < 2020
 	frame->duratonTime = 32000;
-
+	#else
+	frame->durationTime = 32000;
```

Comment:
Was this a typo in the standard previously "duratonTime"?

Reply:
Ha, yeah it was